### PR TITLE
cmd/config: simplify code

### PR DESCRIPTION
**Pull request by geaaru**

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -381,6 +380,10 @@ func (c *LuetConfig) GetSolverOptions() *LuetSolverOptions {
 	return &c.Solver
 }
 
+func (c *LuetConfig) Yaml() ([]byte, error) {
```
Comment (mudler) [inline pkg/config/config.go:383]:
```suggestion
func (c *LuetConfig) YAML() ([]byte, error) {
```

File: cmd/config.go [hunk 2]
```diff
@@ -30,46 +30,12 @@ var configCmd = &cobra.Command{
...
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println(config.LuetCfg.GetLogging())
-		fmt.Println(config.LuetCfg.GetGeneral())
-		fmt.Println(config.LuetCfg.GetSystem())
-		if len(config.LuetCfg.CacheRepositories) > 0 {
-			fmt.Println("repetitors:")
-			for _, r := range config.LuetCfg.CacheRepositories {
-				fmt.Println("  - ", r.String())
-			}
+		data, err := config.LuetCfg.Yaml()
```
Comment (mudler) [inline cmd/config.go:33]:
```suggestion
		data, err := config.LuetCfg.YAML()
```

**Review by mudler — APPROVED** [review]
just a minor nit, but looks good!

File: cmd/config.go (hunk 2 above)
Comment (geaaru) [reply cmd/config.go]: Done ;)

**Pull request merged**